src/test/java/net/datafaker/providers/base/ZodiacTest.java

```diff
@@ -1,14 +1,12 @@
 package net.datafaker.providers.base;
 
+import java.util.Arrays;
+import java.util.Collection;
 
-import org.junit.jupiter.api.Test;
+public class ZodiacTest extends AbstractBasicProviderTest<BaseFaker> {
```

Comment:
```suggestion
class ZodiacTest extends AbstractBasicProviderTest<BaseFaker> {
```
since it is junit5 it could be package private